vipps-report-api.md

```diff
@@ -56,6 +55,7 @@ Vipps APIs as the source of truth for the status of an operation.
 ## Authenticating to the Report API
 
 There are currently two ways to connect to the Report API:
```

Comment:
Link to the FAQ here, maybe? https://vippsas.github.io/vipps-developer-docs/docs/APIs/report-api/vipps-report-api-faq/#which-api-keys-give-access-to-the-api

Reply:
@cloveras - I thought there were 3 ways.... The FAQ shows 3 ways.... ? Shall I add in the _Accounting partners' API keys_

Reply:
Come to think of it ----- wouldn't it be useful to have that information in the guide (and link from the FAQ)? I know we are trying to keep them short, but it is something they need to understand....

Reply:
I'll update this one here and then make another alternative PR that you can look at..